### PR TITLE
Fixes for the manifest generation.

### PR DESCRIPTION
**Pull request by calavera**
1. Create temp files under the default OS temp directory.
2. Remove file extension from manifest, they need to be the name.
3. Serialize JSON in pretty format.

**Checklist**

Please add a `x` inside each checkbox:

- [x] The status checks are successful (continuous integration). Those can be seen below.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -23,26 +24,28 @@ const uploadBundle = require("./upload");
...
     }
 
     const id = "func" + crypto.randomBytes(16).toString("hex");
-    const name = func.name.substr(0, func.name.length - 3); // remove extension //
-    imports += `import * as ${id} from "${path.resolve(src, func.name)}";\n`;
-    registration += `netlifyRegistry.set("${name}", ${id});\n`;
 
-    handlers.push(func.name);
+    imports.push(`import * as ${id} from "${path.resolve(src, file.name)}";`);
+    registration.push(`netlifyRegistry.set("${file.name}", ${id});`);
```
Comment (shortdiv) [inline src/index.js:43]: Nice, this makes it super clear that imports and registration are taking in multiple files

**Review by shortdiv — APPROVED** [review]
LGTM!

**Comment by mraerino** [comment]
> 2\. Remove file extension from manifest, they need to be the name.

i'm not sure what you mean by that. where is this requirement from?
can we make the local mesh proxy use `manifest.json`?

**Comment by calavera** [comment]
>  can we make the local mesh proxy use `manifest.json`?

the previous code generated the list of handlers with the extension, like:

```
handlers: ["hello-world.js"]
```

but it needs to be

```
handlers: ["hello-world"]
```

**Comment by mraerino** [comment]
ah, very good catch!

**Pull request merged**